otus-02/src/otus_02/homework/pangram.clj

```diff
@@ -1,6 +1,17 @@
 (ns otus-02.homework.pangram
-  (:require [clojure.string :as string]))
+  (:require [clojure.string :as string]
+            [clojure.string :as str]))
+(require '[clojure.string :as str])
 
+(def letters
+  [\a \b \c \d \e \f \g \h \i \j \k \l \m \n \o \p \q \r \s \t \u \v \w \x \y \z])
 
-(defn is-pangram [test-string])
+(def chars-set (set (map char (range (int \a) (+ (int \a) 26)))))
+
+(defn is-pangram [test-string]
+  (>= (count (reduce (fn [a e] (conj a e)) (set []) (str/lower-case test-string))) 26)
```

Comment:
`(fn [a e] (conj a e))` == `conj` :)
`(set [])` == `#{}` :)
`(reduce (fn [a e] (conj a e)) (set []) xxx)` == `(set xxx)` :)))

PS: you'll reduce all the string, even if it's already pangram on it's first 2%
PPS: "0123456789!@#$%^&*()_+|~<>,." will be pangram by your code :)